keycloak4s-admin/src/main/scala/com/fullfacing/keycloak4s.admin/services/package.scala

```diff
@@ -15,6 +18,18 @@ package object services {
   type Path = ImmutableSeq[String]
   implicit def uuidToString: UUID => String = _.toString
 
+  def extractStringFromResponse(response: Either[KeycloakError, Headers]): Either[KeycloakError, String] = response.flatMap { headers =>
+    headers
+      .get("Location")
+      .map(location => location.split('/').last)
+      .toRight(Exceptions.ID_NOT_FOUND)
+  }
+
+  def extractUuidFromResponse(response: Either[KeycloakError, Headers]): Either[KeycloakError, UUID] = response.flatMap { headers =>
+    extractStringFromResponse(response)
+      .map(UUID.fromString)
```

Comment:
UUID.fromString can fail (throw a exception), would you like to wrap this side-effecting code?